src/components/reusable/errorBlock/errorBlock.ts

```diff
@@ -0,0 +1,40 @@
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import ErrorScss from './errorBlock.scss';
+
+import '@kyndryl-design-system/shidoka-foundation/components/button';
+
+/**
+ * Error block.
+ * @slot unnamed - Slot for the error description.
+ * @slot image - Slot for the error image.
+ * @slot actions - Slot for the action buttons.
+ */
+
+@customElement('kyn-error-block')
+export class ErrorBlock extends LitElement {
+  static override styles = [ErrorScss];
+
+  /** Title text */
+  @property({ type: String })
+  titleText = '';
+
+  override render() {
+    return html`
+      <div class="error-container">
+        <div class="error-image">
+          <slot name="image"></slot>
+        </div>
+        <div class="error-title">
+          <h1>${this.titleText}</h1>
```

Comment:
Don't hard-code any heading tags. The component will be unaware of the page hierarchy.